src/ProjectNeon/Assets/Features/Effects/DamageApplied.cs

```diff
@@ -0,0 +1,14 @@
+﻿public sealed class DamageApplied : Effect
```

Comment:
Cannot be a past-tense verb yet. When this is created it hasn't happened.

Reply:
@SilasReinagel Made some renaming on `Damage` constructs